sympy/polys/rationaltools.py

```diff
@@ -83,3 +83,65 @@ def _together(expr):
         return expr
 
     return _together(sympify(expr))
+
+
+def thiele_interpolate(u, v, var=symbols('x'), simplify=True):
```

Comment:
Except for the `simplify` option, I think this should mimic the API of `interpolate`:
```
interpolate(data, x)
    Construct an interpolating polynomial for the data points
    evaluated at point x (which can be symbolic or numeric).
```